Error de compilacion

4:03:53 PM: Executing 'build'…

Type-safe project accessors is an incubating feature.
> Task :buildSrc:checkKotlinGradlePluginConfigurationErrors
> Task :buildSrc:compileKotlin UP-TO-DATE
> Task :buildSrc:compileJava NO-SOURCE
> Task :buildSrc:compileGroovy NO-SOURCE
> Task :buildSrc:pluginDescriptors UP-TO-DATE
> Task :buildSrc:processResources NO-SOURCE
> Task :buildSrc:classes UP-TO-DATE
> Task :buildSrc:jar UP-TO-DATE

> Configure project :backend
w: Deprecated Gradle Property 'kotlin.mpp.androidGradlePluginCompatibility.nowarn' Used
The `kotlin.mpp.androidGradlePluginCompatibility.nowarn` deprecated property is used in your build.
Solution: It is unsupported, please stop using it.


> Configure project :app:composeApp
w: JVM Main Run Task Conflict
Target 'desktop': Unable to create run task 'desktopRun' as there is already such a task registered
Solution: Please remove the conflicting task or rename the new task

WARNING: The following problems were found when resolving the SDK location:
Where: sdk.dir property in local.properties file. Problem: Set with empty value


> Task :assemble UP-TO-DATE
> Task :check UP-TO-DATE
> Task :build UP-TO-DATE
> Task :backend:checkKotlinGradlePluginConfigurationErrors SKIPPED
> Task :backend:compileKotlin UP-TO-DATE
> Task :backend:compileJava NO-SOURCE
> Task :backend:processResources UP-TO-DATE
> Task :backend:classes UP-TO-DATE
> Task :backend:jar UP-TO-DATE
> Task :backend:startScripts UP-TO-DATE
> Task :backend:distTar UP-TO-DATE
> Task :backend:distZip UP-TO-DATE
> Task :backend:shadowJar UP-TO-DATE
> Task :backend:startShadowScripts UP-TO-DATE
> Task :backend:shadowDistTar UP-TO-DATE
> Task :backend:shadowDistZip UP-TO-DATE
> Task :backend:assemble UP-TO-DATE
> Task :backend:compileTestKotlin UP-TO-DATE
> Task :backend:compileTestJava NO-SOURCE
> Task :backend:processTestResources NO-SOURCE
> Task :backend:testClasses UP-TO-DATE
> Task :backend:test UP-TO-DATE
> Task :backend:check UP-TO-DATE
> Task :backend:build UP-TO-DATE
> Task :users:checkKotlinGradlePluginConfigurationErrors SKIPPED
> Task :users:compileKotlin UP-TO-DATE
> Task :users:compileJava NO-SOURCE
> Task :users:processResources UP-TO-DATE
> Task :users:classes UP-TO-DATE
> Task :users:jar UP-TO-DATE
> Task :users:startScripts UP-TO-DATE
> Task :users:distTar UP-TO-DATE
> Task :users:distZip UP-TO-DATE
> Task :users:shadowJar UP-TO-DATE
> Task :users:startShadowScripts UP-TO-DATE
> Task :users:shadowDistTar UP-TO-DATE
> Task :users:shadowDistZip UP-TO-DATE
> Task :users:assemble UP-TO-DATE
> Task :users:compileTestKotlin UP-TO-DATE
> Task :users:compileTestJava NO-SOURCE
> Task :users:processTestResources NO-SOURCE
> Task :users:testClasses UP-TO-DATE
> Task :users:test UP-TO-DATE
> Task :users:check UP-TO-DATE
> Task :users:build UP-TO-DATE
> Task :app:composeApp:checkKotlinGradlePluginConfigurationErrors SKIPPED
> Task :app:composeApp:generateBuildKonfig UP-TO-DATE
> Task :app:composeApp:generateComposeResClass UP-TO-DATE
> Task :app:composeApp:generateExpectResourceCollectorsForCommonMain UP-TO-DATE
> Task :app:composeApp:convertXmlValueResourcesForCommonMain
> Task :app:composeApp:copyNonXmlValueResourcesForCommonMain UP-TO-DATE
> Task :app:composeApp:prepareComposeResourcesTaskForCommonMain
> Task :app:composeApp:generateResourceAccessorsForCommonMain
> Task :app:composeApp:transformCommonMainDependenciesMetadata UP-TO-DATE
> Task :app:composeApp:generateProjectStructureMetadata UP-TO-DATE
> Task :app:composeApp:syncBrandingIcons UP-TO-DATE
> Task :app:composeApp:metadataCommonMainProcessResources NO-SOURCE
> Task :app:composeApp:preBuild UP-TO-DATE
> Task :app:composeApp:preDebugBuild UP-TO-DATE
> Task :app:composeApp:mergeDebugNativeDebugMetadata NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForAndroidMain NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAndroidMain NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAndroidMain NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAndroidMain NO-SOURCE
> Task :app:composeApp:generateActualResourceCollectorsForAndroidMain
> Task :app:composeApp:convertXmlValueResourcesForAndroidDebug NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAndroidDebug NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAndroidDebug NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAndroidDebug NO-SOURCE
> Task :app:composeApp:checkDebugAarMetadata UP-TO-DATE
> Task :app:composeApp:generateDebugResValues UP-TO-DATE
> Task :app:composeApp:mapDebugSourceSetPaths UP-TO-DATE
> Task :app:composeApp:generateDebugResources UP-TO-DATE
> Task :app:composeApp:mergeDebugResources UP-TO-DATE
> Task :app:composeApp:packageDebugResources UP-TO-DATE
> Task :app:composeApp:parseDebugLocalResources UP-TO-DATE
> Task :app:composeApp:createDebugCompatibleScreenManifests UP-TO-DATE
> Task :app:composeApp:extractDeepLinksDebug UP-TO-DATE
> Task :app:composeApp:processDebugMainManifest UP-TO-DATE
> Task :app:composeApp:processDebugManifest UP-TO-DATE
> Task :app:composeApp:processDebugManifestForPackage UP-TO-DATE
> Task :app:composeApp:processDebugResources UP-TO-DATE
> Task :app:composeApp:javaPreCompileDebug UP-TO-DATE
> Task :app:composeApp:mergeDebugShaders UP-TO-DATE
> Task :app:composeApp:compileDebugShaders NO-SOURCE
> Task :app:composeApp:copyDebugComposeResourcesToAndroidAssets
> Task :app:composeApp:debugAssetsCopyForAGP NO-SOURCE
> Task :app:composeApp:generateDebugAssets UP-TO-DATE
> Task :app:composeApp:mergeDebugAssets
> Task :app:composeApp:checkDebugDuplicateClasses UP-TO-DATE
> Task :app:composeApp:desugarDebugFileDependencies UP-TO-DATE
> Task :app:composeApp:compressDebugAssets
> Task :app:composeApp:mergeExtDexDebug UP-TO-DATE
> Task :app:composeApp:mergeLibDexDebug UP-TO-DATE
> Task :app:composeApp:mergeDebugJniLibFolders UP-TO-DATE
> Task :app:composeApp:mergeDebugNativeLibs UP-TO-DATE
> Task :app:composeApp:stripDebugDebugSymbols UP-TO-DATE
> Task :app:composeApp:validateSigningDebug UP-TO-DATE
> Task :app:composeApp:writeDebugAppMetadata UP-TO-DATE
> Task :app:composeApp:writeDebugSigningConfigVersions UP-TO-DATE
> Task :app:composeApp:convertXmlValueResourcesForAndroidRelease NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAndroidRelease NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAndroidRelease NO-SOURCE
> Task :app:composeApp:copyReleaseComposeResourcesToAndroidAssets
> Task :app:composeApp:generateResourceAccessorsForAndroidRelease NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForAndroidUnitTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAndroidUnitTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAndroidUnitTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAndroidUnitTest NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForAndroidUnitTestDebug NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAndroidUnitTestDebug NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAndroidUnitTestDebug NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAndroidUnitTestDebug NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForAndroidUnitTestRelease NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAndroidUnitTestRelease NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAndroidUnitTestRelease NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAndroidUnitTestRelease NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForAppleMain NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAppleMain NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAppleMain NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAppleMain NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForAppleTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForAppleTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForAppleTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForAppleTest NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForCommonTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForCommonTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForCommonTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForCommonTest NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForDesktopMain NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForDesktopMain NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForDesktopMain NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForDesktopMain NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForDesktopTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForDesktopTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForDesktopTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForDesktopTest NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForIosMain NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForIosMain NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForIosMain NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForIosMain NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForIosTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForIosTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForIosTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForIosTest NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForIosX64Main NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForIosX64Main NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForIosX64Main NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForIosX64Main NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForIosX64Test NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForIosX64Test NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForIosX64Test NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForIosX64Test NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForNativeMain NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForNativeMain NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForNativeMain NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForNativeMain NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForNativeTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForNativeTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForNativeTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForNativeTest NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForWasmJsMain NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForWasmJsMain NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForWasmJsMain NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForWasmJsMain NO-SOURCE
> Task :app:composeApp:convertXmlValueResourcesForWasmJsTest NO-SOURCE
> Task :app:composeApp:copyNonXmlValueResourcesForWasmJsTest NO-SOURCE
> Task :app:composeApp:prepareComposeResourcesTaskForWasmJsTest NO-SOURCE
> Task :app:composeApp:generateResourceAccessorsForWasmJsTest NO-SOURCE
> Task :app:composeApp:buildKotlinToolingMetadata UP-TO-DATE
> Task :app:composeApp:preReleaseBuild UP-TO-DATE
> Task :app:composeApp:checkReleaseAarMetadata UP-TO-DATE
> Task :app:composeApp:generateReleaseResValues UP-TO-DATE
> Task :app:composeApp:mapReleaseSourceSetPaths UP-TO-DATE
> Task :app:composeApp:generateReleaseResources UP-TO-DATE
> Task :app:composeApp:mergeReleaseResources UP-TO-DATE
> Task :app:composeApp:packageReleaseResources UP-TO-DATE
> Task :app:composeApp:parseReleaseLocalResources UP-TO-DATE
> Task :app:composeApp:createReleaseCompatibleScreenManifests UP-TO-DATE
> Task :app:composeApp:extractDeepLinksRelease UP-TO-DATE
> Task :app:composeApp:processReleaseMainManifest UP-TO-DATE
> Task :app:composeApp:processReleaseManifest UP-TO-DATE
> Task :app:composeApp:processReleaseManifestForPackage UP-TO-DATE
> Task :app:composeApp:processReleaseResources UP-TO-DATE
> Task :app:composeApp:javaPreCompileRelease UP-TO-DATE
> Task :app:composeApp:extractProguardFiles UP-TO-DATE
> Task :app:composeApp:mergeReleaseJniLibFolders UP-TO-DATE
> Task :app:composeApp:mergeReleaseNativeLibs UP-TO-DATE
> Task :app:composeApp:stripReleaseDebugSymbols UP-TO-DATE
> Task :app:composeApp:extractReleaseNativeSymbolTables UP-TO-DATE
> Task :app:composeApp:mergeReleaseNativeDebugMetadata NO-SOURCE
> Task :app:composeApp:checkReleaseDuplicateClasses UP-TO-DATE
> Task :app:composeApp:desugarReleaseFileDependencies UP-TO-DATE
> Task :app:composeApp:mergeReleaseStartupProfile UP-TO-DATE
> Task :app:composeApp:mergeExtDexRelease UP-TO-DATE
> Task :app:composeApp:mergeReleaseArtProfile UP-TO-DATE
> Task :app:composeApp:mergeReleaseShaders UP-TO-DATE
> Task :app:composeApp:compileReleaseShaders NO-SOURCE
> Task :app:composeApp:generateReleaseAssets UP-TO-DATE
> Task :app:composeApp:releaseAssetsCopyForAGP NO-SOURCE
> Task :app:composeApp:mergeReleaseAssets
> Task :app:composeApp:extractReleaseVersionControlInfo
> Task :app:composeApp:optimizeReleaseResources UP-TO-DATE
> Task :app:composeApp:collectReleaseDependencies UP-TO-DATE
> Task :app:composeApp:sdkReleaseDependencyData UP-TO-DATE
> Task :app:composeApp:writeReleaseAppMetadata UP-TO-DATE
> Task :app:composeApp:writeReleaseSigningConfigVersions UP-TO-DATE
> Task :app:composeApp:generateActualResourceCollectorsForIosX64Main
> Task :app:composeApp:compressReleaseAssets
> Task :app:composeApp:compileKotlinIosX64 SKIPPED
> Task :app:composeApp:generateActualResourceCollectorsForWasmJsMain
> Task :app:composeApp:wasmJsResolveResourcesFromDependencies UP-TO-DATE
> Task :app:composeApp:assembleWasmJsMainResources
> Task :app:composeApp:wasmJsResolveSelfResourcesCopyHierarchicalMultiplatformResources
> Task :app:composeApp:wasmJsAggregateResources
> Task :app:composeApp:wasmJsProcessResources
> Task :app:composeApp:kotlinWasmBinaryenSetup UP-TO-DATE
> Task :app:composeApp:unpackSkikoWasmRuntime UP-TO-DATE
> Task :app:composeApp:processSkikoRuntimeForKWasm UP-TO-DATE
> Task :app:composeApp:iosX64ResolveResourcesFromDependencies UP-TO-DATE
> Task :app:composeApp:assembleIosX64MainResources
> Task :app:composeApp:iosX64ResolveSelfResourcesCopyHierarchicalMultiplatformResources
> Task :app:composeApp:iosX64AggregateResources
> Task :app:composeApp:iosX64ProcessResources
> Task :app:composeApp:iosX64MainKlibrary
> Task :app:composeApp:compileTestKotlinIosX64 SKIPPED
> Task :app:composeApp:generateActualResourceCollectorsForDesktopMain
> Task :app:composeApp:assembleDesktopMainResources
> Task :app:composeApp:desktopProcessResources
> Task :app:composeApp:processDesktopMainResources SKIPPED
> Task :app:composeApp:iosX64MetadataElements SKIPPED
> Task :app:composeApp:linkDebugFrameworkIosX64 SKIPPED
> Task :app:composeApp:linkReleaseFrameworkIosX64 SKIPPED
> Task :kotlinWasmKotlinNpmCachesSetup
> Task :kotlinWasmNodeJsSetup UP-TO-DATE
> Task :kotlinWasmRestoreYarnLock UP-TO-DATE
> Task :kotlinWasmYarnSetup UP-TO-DATE
> Task :app:composeApp:wasmJsPackageJson UP-TO-DATE
> Task :app:composeApp:wasmJsTestPackageJson UP-TO-DATE
> Task :app:composeApp:wasmJsPublicPackageJson UP-TO-DATE
> Task :app:composeApp:wasmJsTestPublicPackageJson UP-TO-DATE
> Task :kotlinWasmPackageJsonUmbrella UP-TO-DATE
> Task :wasmRootPackageJson UP-TO-DATE
> Task :kotlinWasmNpmInstall UP-TO-DATE
> Task :kotlinWasmStoreYarnLock UP-TO-DATE
> Task :app:composeApp:assembleDesktopTestResources UP-TO-DATE
> Task :app:composeApp:desktopTestProcessResources UP-TO-DATE
> Task :app:composeApp:processDesktopTestResources SKIPPED
> Task :app:composeApp:assembleIosX64TestResources UP-TO-DATE
> Task :app:composeApp:copyTestComposeResourcesForIosX64
> Task :app:composeApp:linkDebugTestIosX64 SKIPPED
> Task :app:composeApp:iosX64Test SKIPPED
> Task :app:composeApp:preDebugUnitTestBuild UP-TO-DATE
> Task :app:composeApp:javaPreCompileDebugUnitTest UP-TO-DATE
> Task :app:composeApp:preReleaseUnitTestBuild UP-TO-DATE
> Task :app:composeApp:javaPreCompileReleaseUnitTest UP-TO-DATE
> Task :app:composeApp:kotlinWasmNodeJsSetup UP-TO-DATE
> Task :app:composeApp:assembleWasmJsTestResources UP-TO-DATE
> Task :app:composeApp:wasmJsTestProcessResources UP-TO-DATE
> Task :app:composeApp:preDebugAndroidTestBuild SKIPPED
> Task :app:composeApp:generateDebugAndroidTestResValues UP-TO-DATE
> Task :app:composeApp:compileCommonMainKotlinMetadata
> Task :app:composeApp:compileDebugKotlinAndroid
> Task :app:composeApp:compileReleaseKotlinAndroid

> Task :app:composeApp:compileKotlinWasmJs FAILED
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:41:34 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:246:47 @Composable invocations can only happen from the context of a @Composable function
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:364:18 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:374:22 Unresolved reference 'clickable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:56:59 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:78:56 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:99:57 Unresolved reference 'semi_circular_menu_open'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:100:58 Unresolved reference 'semi_circular_menu_close'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:101:46 Unresolved reference 'dashboard_menu_hint'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:189:61 Unresolved reference 'buttons_preview'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:190:61 Unresolved reference 'change_password'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:191:61 Unresolved reference 'two_factor_setup'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:192:62 Unresolved reference 'two_factor_verify'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:193:63 Unresolved reference 'register_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:194:58 Unresolved reference 'request_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:195:61 Unresolved reference 'review_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:196:57 Unresolved reference 'review_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:197:60 Unresolved reference 'register_saler'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:198:53 Unresolved reference 'logout'.

> Task :app:composeApp:compileReleaseKotlinAndroid FAILED
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:41:34 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:246:47 @Composable invocations can only happen from the context of a @Composable function
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:364:18 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:374:22 Unresolved reference 'clickable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:56:59 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:78:56 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:99:57 Unresolved reference 'semi_circular_menu_open'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:100:58 Unresolved reference 'semi_circular_menu_close'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:101:46 Unresolved reference 'dashboard_menu_hint'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:189:61 Unresolved reference 'buttons_preview'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:190:61 Unresolved reference 'change_password'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:191:61 Unresolved reference 'two_factor_setup'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:192:62 Unresolved reference 'two_factor_verify'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:193:63 Unresolved reference 'register_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:194:58 Unresolved reference 'request_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:195:61 Unresolved reference 'review_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:196:57 Unresolved reference 'review_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:197:60 Unresolved reference 'register_saler'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:198:53 Unresolved reference 'logout'.

> Task :app:composeApp:compileCommonMainKotlinMetadata
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:41:34 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:246:47 @Composable invocations can only happen from the context of a @Composable function
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:364:18 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:374:22 Unresolved reference 'clickable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:56:59 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:78:56 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:99:57 Unresolved reference 'semi_circular_menu_open'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:100:58 Unresolved reference 'semi_circular_menu_close'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:101:46 Unresolved reference 'dashboard_menu_hint'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:189:61 Unresolved reference 'buttons_preview'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:190:61 Unresolved reference 'change_password'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:191:61 Unresolved reference 'two_factor_setup'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:192:62 Unresolved reference 'two_factor_verify'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:193:63 Unresolved reference 'register_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:194:58 Unresolved reference 'request_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:195:61 Unresolved reference 'review_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:196:57 Unresolved reference 'review_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:197:60 Unresolved reference 'register_saler'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:198:53 Unresolved reference 'logout'.

> Task :app:composeApp:compileCommonMainKotlinMetadata FAILED
> Task :app:composeApp:compileKotlinDesktop
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:41:34 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:246:47 @Composable invocations can only happen from the context of a @Composable function
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:364:18 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:374:22 Unresolved reference 'clickable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:56:59 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:78:56 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:99:57 Unresolved reference 'semi_circular_menu_open'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:100:58 Unresolved reference 'semi_circular_menu_close'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:101:46 Unresolved reference 'dashboard_menu_hint'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:189:61 Unresolved reference 'buttons_preview'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:190:61 Unresolved reference 'change_password'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:191:61 Unresolved reference 'two_factor_setup'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:192:62 Unresolved reference 'two_factor_verify'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:193:63 Unresolved reference 'register_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:194:58 Unresolved reference 'request_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:195:61 Unresolved reference 'review_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:196:57 Unresolved reference 'review_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:197:60 Unresolved reference 'register_saler'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:198:53 Unresolved reference 'logout'.

> Task :app:composeApp:compileKotlinDesktop FAILED

> Task :app:composeApp:compileDebugKotlinAndroid
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:41:34 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:246:47 @Composable invocations can only happen from the context of a @Composable function
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:364:18 Unresolved reference 'focusable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt:374:22 Unresolved reference 'clickable'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:56:59 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:78:56 Unresolved reference 'dashboard'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:99:57 Unresolved reference 'semi_circular_menu_open'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:100:58 Unresolved reference 'semi_circular_menu_close'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:101:46 Unresolved reference 'dashboard_menu_hint'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:189:61 Unresolved reference 'buttons_preview'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:190:61 Unresolved reference 'change_password'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:191:61 Unresolved reference 'two_factor_setup'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:192:62 Unresolved reference 'two_factor_verify'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:193:63 Unresolved reference 'register_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:194:58 Unresolved reference 'request_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:195:61 Unresolved reference 'review_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:196:57 Unresolved reference 'review_join_business'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:197:60 Unresolved reference 'register_saler'.
e: file:///C:/Workspaces/Intrale/platform/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt:198:53 Unresolved reference 'logout'.

> Task :app:composeApp:compileDebugKotlinAndroid FAILED

FAILURE: Build completed with 5 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:composeApp:compileKotlinWasmJs'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:composeApp:compileReleaseKotlinAndroid'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:composeApp:compileCommonMainKotlinMetadata'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

4: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:composeApp:compileKotlinDesktop'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

5: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:composeApp:compileDebugKotlinAndroid'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

BUILD FAILED in 51s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.9/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
146 actionable tasks: 32 executed, 114 up-to-date
4:04:49 PM: Execution finished 'build'.

--- a/app/composeApp/build.gradle.kts
+++ b/app/composeApp/build.gradle.kts
@@ -63,6 +63,10 @@ kotlin {
     }
     
     sourceSets {
+        val commonMain by getting {
+            val generatedSources = layout.buildDirectory.dir("generated/compose/resourceGenerator/kotlin")
+            kotlin.srcDir(generatedSources)
+        }
         val desktopMain by getting
 
         androidMain.dependencies {

--- a/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt
+++ b/app/composeApp/src/commonMain/kotlin/ui/cp/menu/SemiCircularHamburgerMenu.kt
@@ -6,8 +6,8 @@ import androidx.compose.animation.core.animateDpAsState
 import androidx.compose.animation.core.animateFloatAsState
 import androidx.compose.animation.core.spring
 import androidx.compose.animation.core.tween
-import androidx.compose.foundation.Canvas
 import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
 import androidx.compose.foundation.gestures.detectDragGestures
 import androidx.compose.foundation.gestures.detectTapGestures
 import androidx.compose.foundation.interaction.MutableInteractionSource
@@ -37,8 +37,9 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.drawBehind
 import androidx.compose.ui.draw.rotate
-import androidx.compose.ui.focus.focusable
+import androidx.compose.foundation.focusable
 import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.geometry.Size
 import androidx.compose.ui.graphics.Color
@@ -239,13 +240,18 @@ fun SemiCircularHamburgerMenu(
                         }
                     }
             ) {
-                Canvas(modifier = Modifier.matchParentSize()) {
-                    drawSemiCircle(
-                        sweepAngle = sweepAngle.coerceIn(0f, arcDegrees),
-                        startAngle = startAngleDegrees,
-                        color = MaterialTheme.colorScheme.primary
-                    )
-                }
+                val arcColor = MaterialTheme.colorScheme.primary
+                Box(
+                    modifier = Modifier
+                        .matchParentSize()
+                        .drawBehind {
+                            drawSemiCircle(
+                                sweepAngle = sweepAngle.coerceIn(0f, arcDegrees),
+                                startAngle = startAngleDegrees,
+                                color = arcColor
+                            )
+                        }
+                )
 
                 MenuToggleIcon(
                     expanded = expanded,

--- a/app/composeApp/src/commonMain/kotlin/ui/rs/DashboardStrings.kt
+++ b/app/composeApp/src/commonMain/kotlin/ui/rs/DashboardStrings.kt
@@ -1,0 +1,151 @@
+@file:OptIn(InternalResourceApi::class)
+
+package ui.rs
+
+import kotlin.OptIn
+import kotlin.collections.setOf
+import org.jetbrains.compose.resources.InternalResourceApi
+import org.jetbrains.compose.resources.ResourceItem
+import org.jetbrains.compose.resources.StringResource
+
+private const val RESOURCE_PREFIX = "composeResources/ui.rs/"
+
+val dashboard: StringResource by lazy {
+    StringResource(
+        "string:dashboard",
+        "dashboard",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 543, 37)
+        )
+    )
+}
+
+val semi_circular_menu_open: StringResource by lazy {
+    StringResource(
+        "string:semi_circular_menu_open",
+        "semi_circular_menu_open",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 3077, 63)
+        )
+    )
+}
+
+val semi_circular_menu_close: StringResource by lazy {
+    StringResource(
+        "string:semi_circular_menu_close",
+        "semi_circular_menu_close",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 3012, 64)
+        )
+    )
+}
+
+val dashboard_menu_hint: StringResource by lazy {
+    StringResource(
+        "string:dashboard_menu_hint",
+        "dashboard_menu_hint",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 435, 107)
+        )
+    )
+}
+
+val buttons_preview: StringResource by lazy {
+    StringResource(
+        "string:buttons_preview",
+        "buttons_preview",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 236, 55)
+        )
+    )
+}
+
+val change_password: StringResource by lazy {
+    StringResource(
+        "string:change_password",
+        "change_password",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 292, 51)
+        )
+    )
+}
+
+val two_factor_setup: StringResource by lazy {
+    StringResource(
+        "string:two_factor_setup",
+        "two_factor_setup",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 3404, 76)
+        )
+    )
+}
+
+val two_factor_verify: StringResource by lazy {
+    StringResource(
+        "string:two_factor_verify",
+        "two_factor_verify",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 3481, 77)
+        )
+    )
+}
+
+val register_business: StringResource by lazy {
+    StringResource(
+        "string:register_business",
+        "register_business",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 2352, 49)
+        )
+    )
+}
+
+val request_join_business: StringResource by lazy {
+    StringResource(
+        "string:request_join_business",
+        "request_join_business",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 2666, 53)
+        )
+    )
+}
+
+val review_business: StringResource by lazy {
+    StringResource(
+        "string:review_business",
+        "review_business",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 2720, 79)
+        )
+    )
+}
+
+val review_join_business: StringResource by lazy {
+    StringResource(
+        "string:review_join_business",
+        "review_join_business",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 2900, 68)
+        )
+    )
+}
+
+val register_saler: StringResource by lazy {
+    StringResource(
+        "string:register_saler",
+        "register_saler",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 2477, 46)
+        )
+    )
+}
+
+val logout: StringResource by lazy {
+    StringResource(
+        "string:logout",
+        "logout",
+        setOf(
+            ResourceItem(setOf(), "${RESOURCE_PREFIX}values/strings.commonMain.cvr", 2019, 22)
+        )
+    )
+}

--- a/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt
+++ b/app/composeApp/src/commonMain/kotlin/ui/sc/business/DashboardScreen.kt
@@ -40,7 +40,20 @@ import ui.cp.buttons.Button
 import ui.cp.menu.MainMenuItem
 import ui.cp.menu.MenuState
 import ui.cp.menu.SemiCircularHamburgerMenu
-import ui.rs.Res
+import ui.rs.buttons_preview
+import ui.rs.change_password
+import ui.rs.dashboard
+import ui.rs.dashboard_menu_hint
+import ui.rs.logout
+import ui.rs.register_business
+import ui.rs.register_saler
+import ui.rs.request_join_business
+import ui.rs.review_business
+import ui.rs.review_join_business
+import ui.rs.semi_circular_menu_close
+import ui.rs.semi_circular_menu_open
+import ui.rs.two_factor_setup
+import ui.rs.two_factor_verify
 import ui.th.spacing
 import ui.sc.auth.CHANGE_PASSWORD_PATH
 import ui.sc.auth.TWO_FACTOR_SETUP_PATH
@@ -53,7 +66,7 @@ import ui.sc.signup.REGISTER_SALER_PATH
 const val DASHBOARD_PATH = "/dashboard"
 private const val ENABLE_SEMI_CIRCULAR_MENU = true
 
-class DashboardScreen : Screen(DASHBOARD_PATH, Res.string.dashboard) {
+class DashboardScreen : Screen(DASHBOARD_PATH, dashboard) {
 
     private val logger = LoggerFactory.default.newLogger<DashboardScreen>()
 
@@ -75,7 +88,7 @@ class DashboardScreen : Screen(DASHBOARD_PATH, Res.string.dashboard) {
                 item.requiredRoles.isEmpty() || currentUserRole?.let { role -> role in item.requiredRoles } == true
             }
         }
-        val dashboardTitle = stringResource(Res.string.dashboard)
+        val dashboardTitle = stringResource(dashboard)
 
         if (ENABLE_SEMI_CIRCULAR_MENU) {
             DashboardMenuWithSemiCircle(
@@ -96,9 +109,9 @@ class DashboardScreen : Screen(DASHBOARD_PATH, Res.string.dashboard) {
         items: List<MainMenuItem>,
         title: String,
     ) {
-        val openDescription = stringResource(Res.string.semi_circular_menu_open)
-        val closeDescription = stringResource(Res.string.semi_circular_menu_close)
-        val hint = stringResource(Res.string.dashboard_menu_hint)
+        val openDescription = stringResource(semi_circular_menu_open)
+        val closeDescription = stringResource(semi_circular_menu_close)
+        val hint = stringResource(dashboard_menu_hint)
 
         Box(
             modifier = Modifier
@@ -186,16 +199,16 @@ class DashboardScreen : Screen(DASHBOARD_PATH, Res.string.dashboard) {
         viewModel: DashboardViewModel,
         coroutineScope: CoroutineScope
     ): List<MainMenuItem> {
-        val buttonsPreviewLabel = stringResource(Res.string.buttons_preview)
-        val changePasswordLabel = stringResource(Res.string.change_password)
-        val setupTwoFactorLabel = stringResource(Res.string.two_factor_setup)
-        val verifyTwoFactorLabel = stringResource(Res.string.two_factor_verify)
-        val registerBusinessLabel = stringResource(Res.string.register_business)
-        val requestJoinLabel = stringResource(Res.string.request_join_business)
-        val reviewBusinessLabel = stringResource(Res.string.review_business)
-        val reviewJoinLabel = stringResource(Res.string.review_join_business)
-        val registerSalerLabel = stringResource(Res.string.register_saler)
-        val logoutLabel = stringResource(Res.string.logout)
+        val buttonsPreviewLabel = stringResource(buttons_preview)
+        val changePasswordLabel = stringResource(change_password)
+        val setupTwoFactorLabel = stringResource(two_factor_setup)
+        val verifyTwoFactorLabel = stringResource(two_factor_verify)
+        val registerBusinessLabel = stringResource(register_business)
+        val requestJoinLabel = stringResource(request_join_business)
+        val reviewBusinessLabel = stringResource(review_business)
+        val reviewJoinLabel = stringResource(review_join_business)
+        val registerSalerLabel = stringResource(register_saler)
+        val logoutLabel = stringResource(logout)
 
         return remember(
             buttonsPreviewLabel,

